### ISSUE TITLE
My Site more menu accessibility

### ISSUE THREAD

**Issue by nbradbury** [wordpress-mobile/WordPress-Android#21409]
As mentioned [here](https://github.com/wordpress-mobile/WordPress-Android/pull/21383#pullrequestreview-2413182465):

> One consideration for a tangential UI button: the menu button (three vertical dots) on the My Site tab lacks contextual information. I.e., a description of "more" requires one to know they are within the "prompts" card, where a description of "more prompts" or "view more prompts" would mitigate any confusion. There are also several "more" buttons on the screen, further reducing clarity. We might consider updating the button description to reduce the cognitive load for users relying upon screen readers.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
